packages/nuxt/src/app/components/nuxt-root.vue

```diff
@@ -13,6 +13,7 @@
       :is="SingleRenderer"
       v-else-if="SingleRenderer"
     />
+    <LazyNuxtPage v-else-if="route.meta?.isolate" />
```

Comment:
this implementation will result in always bundling `<NuxtPage>`, even if the `vue-router` integration isn't enabled.

we should probably protect it so that if no routes have the `isolate` metadata then there will be no changes to the bundle at all